02_calculator/calculator.rb

```diff
@@ -1 +1,50 @@
-#write your code here
+
+
+def add ( num1,num2)
+
+   
+  add=num1 + num2
+end
+add(0,0)
+
+
+def subtract ( num1,num2)
+
+   
+  subtract=num1 - num2
+    end
+subtract(10,4)
+
+    
+    
+  
+
+
+def sum array
+  sum = 0
+  array.map do |number| 
+    sum+= number
+  end
+  sum
+end
+
+def multiply *array
+ array.reduce(:*)
+end
+
+puts multiply(3,9)
+puts multiply(1,2,3,4)
```

Comment:
You really don't need to put these here; the objective of this lesson is to get you into the habit of writing the **test first** and then evaluating **using the test**.